rust/operator-binary/src/hbase_controller.rs

```diff
@@ -1,4 +1,6 @@
-//! Ensures that `Pod`s are configured and running for each [`HbaseCluster`]
+//! Ensures that `Pod`s are configured and running for each [`HbaseCluster`][v1alpha1]
+//!
+//! [v1alpha1]: crate::crd::v1alpha1::HbaseCluster
```

Comment:
suggestion: specify the version in the docs 9to make it more obvious if the parameters change version.

```suggestion
//! Ensures that `Pod`s are configured and running for each [`v1alpha1::HbaseCluster`]
```